package/functions/funcs/voiceMembersCount.js

```diff
@@ -0,0 +1,16 @@
+module.exports = async (d) => {
+  const code = d.command.code;
+  const inside = d.unpack();
+  let result;
+  const [guildID = d.message.guild.id] = inside.splits;
+  
+  const guild = d.client.guilds.cache.get(guildID)
+  
+  if(!guild) return d.error(`\`$voiceMembersCount: Invalid guild ID in ${inside}\``)
```

Comment:
@berat141 instead of using the function name directly in d.error use `${d.func}` instead because it removes possibility of future changes if the function name was changed